docs/usage/bicep.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Bicep
+description: Bicep dependencies support in Renovate
+---
+
+# Bicep
+
+Renovate supports upgrading API versions in `resource` references.
+Upgrading `module` versions is not supported.
```

Comment:
I've not used Bicep modules, but investigating it a little bit it looks like they're [stored in container registries][1] :open_mouth:

Seems like it should be possible to do this, if someone wanted to implement it later

[1]: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/modules#file-in-registry